reledmac.dtx

```diff
@@ -10518,7 +10662,7 @@
 % are called \protect\cs{vAfootnote}, \protect\cs{Afootfmt}, \protect\cs{Afootstart},
 % and \protect\cs{Afootgroup}.
 %
-% These macros are changed depending of the footnotes arrangement: ``normal'', ``paragraphed'', ``two columns'' or ``three columns''.
+% These macros are changed depending on the footnotes arrangement: ``normal'', ``paragraphed'', ``two columns'' or ``three columns''.
```

Comment:
"footnotes" → "footnote"

Reply:
why singular? There are many footnotes.

Reply:
Because it's the singular "footnote arrangement" (like one "flower arrangement" of many flowers). You could also say "depending on the arrangement of the footnotes".

Reply:
oh yes, I see! "Footnote" becomes, in some way, an adjective.

Reply:
Yes, more or less.